File: ITCR.InclusionesTec/ITCR.CommonClasses/Horario.cs
#region Acerca de...
/////////////////////////////////////////////////////////////////////////////
//  Instituto Tecnológico de Costa Rica
//  Proyecto: Inclutec - Sistema Automatizado de Inclusiones
//  Descripción: Clase entidad para la tabla Horario
//  Generado por ITCR Gen v1.0.0.0 
//  Fecha: Viernes, 3 de Enero de 2014, 11:08:00 a.m.
////////////////////////////////////////////////////////////////////////////
#endregion

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace ITCR.Ado.ClasesComunes
{
    public class Horario
    {        
        #region Atributos
        #endregion

        #region Constructores
        #endregion

        #region Metodos
        #endregion

        #region Propiedades

        /// <summary>
        /// Corresponde al identificador de horario
        /// </summary>
        public int Id_Horario { get; set; }

        /// <summary>
        /// Corresponde al dia del horario
        /// </summary>
        public string Txt_Dia { get; set; }

        /// <summary>
        /// Corresponde a la hora de inicio
        /// </summary>
        public string Txt_Hora_Inicio { get; set; }

        /// <summary>
        /// Corresponde a la hora de final
        /// </summary>
        public string Txt_Hora_Final { get; set; }

        #endregion
    }
}

File: ITCR.Inclutec/src/com/itcr/inclutec/clases_comunes/Horario.java
package com.itcr.inclutec.clases_comunes;

public class Horario {
	public int Id_Horario;
    public String Txt_Dia, Txt_Hora_Inicio, Txt_Hora_Final;
	public int getId_Horario() {
		return Id_Horario;
	}
	public void setId_Horario(int id_Horario) {
		Id_Horario = id_Horario;
	}
	public String getTxt_Dia() {
		return Txt_Dia;
	}
	public void setTxt_Dia(String txt_Dia) {
		Txt_Dia = txt_Dia;
	}
	public String getTxt_Hora_Inicio() {
		return Txt_Hora_Inicio;
	}
	public void setTxt_Hora_Inicio(String txt_Hora_Inicio) {
		Txt_Hora_Inicio = txt_Hora_Inicio;
	}
	public String getTxt_Hora_Final() {
		return Txt_Hora_Final;
	}
	public void setTxt_Hora_Final(String txt_Hora_Final) {
		Txt_Hora_Final = txt_Hora_Final;
	}
}
